extensions/binding/org.eclipse.smarthome.binding.weatherunderground/README.md

```diff
@@ -16,22 +16,28 @@ To use this binding, you first need to [register and get your API key](https://w
 
 ## Supported Things
 
-There is exactly two supported things type. The first one is the weather thing, which represents the weather information for an observation location. It has the id `weather`. The second one is the bridge thing, which allows one to add an API key only one time which all sub-sequents weather things will use. It has the id `bridge
+There are exactly two supported things type. The first one is the bridge thing, which represents the connection to the Weather Underground service through the API key. It has the id `bridge`. The second one is the weather thing, which represents the weather information for an observed location. It has the id `weather`.  Each `weather` thing uses a `bridge` thing ; it cannot be set online if no `bridge` thing is defined.
 
 ## Discovery
 
 If a system location is set, "Local Weather" will be automatically discovered for this location.
 
 If the system location is changed, the background discovery updates the configuration of "Local Weather" automatically.
 
-After adding this discovered thing, you will have to set the correct API key. 
+If a bridge is correctly configured, the discovered thing will automatically go online. 
 
 ## Binding Configuration
 
 The binding has no configuration options, all configuration is done at Thing and Channel levels.
 
 ## Thing Configuration
 
+The bridge only has one configuration parameter:
```

Comment:
Just for verification: only has or has only ?

Reply:
Both are correct, I just think the first one sounds better. Also the one I heard English speakers say the most.